src/SoundList.js

```diff
@@ -3,10 +3,24 @@ import { Link } from "react-router-dom";
 import { playCircleSharp, timeSharp, downloadSharp } from "ionicons/icons";
 import { IonIcon } from "@ionic/react";
 
-function timeConvert(minutes) {
-  const min = Math.floor(Math.abs(minutes));
-  const secs = Math.floor((Math.abs(minutes) * 60) % 60);
-  return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
+function convertSecondsInDecimalToHHMMSSFormat(secondsInDecimal) {
```

Comment:
Any ideas for better names? I recommend taking a look at the date-fns library and seeing what their API naming looks like

Reply:
I changed it to `formatTime`.